framework/source/dom/Dispatcher.js

```diff
@@ -30,10 +30,11 @@ enyo.dispatcher = {
 	windowEvents: ["resize", "load", "unload"],
 	connect: function() {
 		var d = enyo.dispatcher;
-		for (var i=0, e; e=d.events[i]; i++) {
+		var e;
+		for (i=0, e; e=d.events[i]; i++) {
```

Comment:
this will probably not work, "i" is not declared before the for loop; just do "var i,e" in the previous line and it should be ok then